src/main/java/com/kkm/kkm_server_v2/domian/auth/presentation/KakaoController.java

```diff
@@ -0,0 +1,42 @@
+package com.kkm.kkm_server_v2.domian.auth.presentation;
+import com.kkm.kkm_server_v2.domian.auth.presentation.dto.request.LoginRequest;
+import com.kkm.kkm_server_v2.domian.auth.presentation.dto.response.AccessTokenResponse;
+import com.kkm.kkm_server_v2.domian.auth.service.KakaoAuthService;
+import com.kkm.kkm_server_v2.domian.auth.service.LoginService;
+import com.kkm.kkm_server_v2.domian.auth.service.RefreshTokenService;
+import com.kkm.kkm_server_v2.domian.user.facade.UserFacade;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/auth")
+@RequiredArgsConstructor
+public class KakaoController {
```

Comment:
```suggestion
@RequestMapping("/auth/kakao")
@RequiredArgsConstructor
public class KakaoController {
```

이렇게 해서 Url을 한 번에 지정해줘도 좋습니다

Reply:
좋은 생각입니다.!